Lib/multiprocessing/util.py

```diff
@@ -102,6 +102,27 @@ def log_to_stderr(level=None):
     _log_to_stderr = True
     return _logger
 
+
+# Abstract socket support
+
+def _platform_supports_abstract_sockets():
+    if "linux" in sys.platform:
+        return True
+    if hasattr(sys, 'getandroidapilevel'):
+        return True
+    return False
+
+
+def is_abstract_socket_namespace(address):
+    if isinstance(address, bytes):
+        return address[0] == 0
+    elif isinstance(address, str):
+        return address[0] == "\0"
+    raise ValueError('address type of %r unrecognized' % address)
```

Comment:
```suggestion
    raise TypeError('address type of {address!r} unrecognized')
```

`%r` causes issues if address is a tuple.